### ISSUE TITLE
The validation process of MSCOCO

### ISSUE THREAD

**Issue by zwyang6** [rulixiang/ToCo#10]
Hi ！ Thanks for your briliant work in WSSS. I notice that you only use part of cases of the val dataset (i.e., 5000 out of 40137) and it only takes almost 18 mins to validate the 5000 val cases. I guess it is an efficient alternative to validate the efficacy of your model  by only using part of val cases . But I am wondering is it fine to do so ?

**Comment by rulixiang** [comment]
Hi, Thanks for your interest. The results in the paper are evaluated on the full validation set (~40k images). I just use 5000 images for fast validation.

**Comment by zwyang6** [comment]
Thanks! it that means you additionally use the saved checkpoints to evaluate on the full validation set after training? or direcly do that during training

**Comment by rulixiang** [comment]
Yes, I used the saved checkpoints to evaluate the full validation set after training. Evaluating the full set during training is too time-consuming.

**Comment by zwyang6** [comment]
Thanks! It is a smart try and it sounds great !
